programs/spl-token-collective/src/account.rs

```diff
@@ -68,6 +68,29 @@ pub struct InitializeCollectiveV0<'info> {
   pub rent: Sysvar<'info, Rent>,
 }
 
+#[derive(Accounts)]
+#[instruction(args: InitializeCollectiveForSocialTokenV0Args)]
+pub struct InitializeCollectiveForSocialTokenV0<'info> {
+  #[account(init, seeds = [
+    b"collective", 
+    mint.key().as_ref()],
+    payer=payer,
+    bump,
+    space=312
+  )]
+  pub collective: Box<Account<'info, CollectiveV0>>,
+  pub mint: Box<Account<'info, Mint>>,
+  #[account(
+    constraint = owner_token_ref.authority.unwrap() == payer.key() || mint.mint_authority.unwrap() == payer.key()
+  )]
+  pub owner_token_ref: Box<Account<'info, TokenRefV0>>,
```

Comment:
No seeds check, so this really could be any token_ref. mint token ref, owner token ref, etc. Can probably just call it `token_ref`.